src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] color = Color.values();
+        Random generator = new Random();
```

Comment:
давай вынесем переменную Random на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод getRandomColor().
```suggestion
        Random random = new Random();
```